### ISSUE TITLE
Cross browser issues

### ISSUE THREAD

**Issue by jrosindell** [OneZoom/OZtree#247]
- [ ] Firefox (windows) on museum display has strange CSS bug for location feature where the menu does not line up with the others
- [ ] Firefox and Edge (windows) rescale the entire page rather than reading multi touch inputs as zoom commands on the page itself.

**Comment by hyanwong** [comment]
Are these things to ask Matt or Jamie to fix?

**Comment by lentinj** [comment]
> Firefox (windows) on museum display has strange CSS bug for location feature where the menu does not line up with the others

And Linux. It's the locationDropdown element getting in the way of the icon-container and label. Moving it to after the label seems to help.

**Comment by lentinj** [comment]
> multi touch inputs

multi-touch touchpad, or touchscreen? Presumably the latter.

**Comment by jrosindell** [comment]
Yes, it's touchscreen that doesn't work - though I'm talking about a Windows 10 PC with touchscreen, not a mobile touch device.

**Comment by hyanwong** [comment]
I've recently changed some of the control buttons on the museum display (life_MDtouch and friends), but don't think this bug should be affected. Worth syncing with master before debugging though.

**Comment by lentinj** [comment]
Commit 32cf5b0 (in work-2020-04-21) I think will fix the latter half of this issue. Basically a CSS property designed with us in mind, disable any default handling of touch events so we do it.

This seems to work in Firefox under my testing VM with the touchscreen passed through to the VM. However Edge doesn't recognise pinching, not sure if this is a local issue or not though.

**Comment by jrosindell** [comment]
Looks good - I'll leave the issue open for later - when the lockdown ends I'll test it on some other machines and see if the touch works or not.

**Comment by jrosindell** [comment]
I'm just adding a note here as there are reports from the OZ FaceBook page that the front page does not load for some users.  I'm trying to get some more info on this.

"Internal Error". The site doesn´t work.
Two different PC:s. Edge, Firefox IE and Chrome. Same result.
I live in Sweden if that is relevant for some reason.

the same for me from France, but it works here http://www.onezoom.org/life.html/@Mammalia?init=zoom

**Comment by hyanwong** [comment]
Look at the logs. Have you got the time they visited?

**Comment by lentinj** [comment]
HTTP vs HTTPS might be interesting too (admittedly unlikely nowadays, but maybe their ISP still has some attempts at HTML/JS rewriting turned on).
